spec.md

```diff
@@ -546,6 +548,13 @@ The tags MUST be in lexical order.
 
 When using the `last` query parameter, the `n` parameter is OPTIONAL.
 
+If the registry is unable to return all tags requested by the client, the registry MUST return a `413 Content Too Large` with `TAG_LIMIT_EXCEEDED` error <sup>[code-15](#error-codes)</sup> and include the header `OCI-Tag-Limit: <limit>`, where `<limit>` is the maximum tags per request supported by the registry.
+
+*Implementers note:*
+Most registries support at least 1000 tags per request, but limits have been seen as low as 50 tags per request.
+To maximize the tags per page, a request without setting `n` can be used, which should either return the full list of tags, or an error with the `OCI-Tag-Limit` header indicating the limit.
+When available, clients should prefer the `Link` header over using the `last` parameter for pagination.
```

Comment:
I would promote this from implementers note to the actual spec language somewhere near this line:

> A `Link` header MAY be included in the response when additional tags are available.

Reply:
I updated line 532 and reworded these notes.